src/lib.rs

```diff
@@ -37,6 +37,8 @@ pub struct Interface {
     pub address: std::net::IpAddr,
     /// The netmask of the interface, if available.
     pub netmask: Option<std::net::IpAddr>,
+    /// The destination of the interface, if available.
```

Comment:
I think this needs slightly more refinement -- it's valid for netbsd and darwin, AFAICT and the field should ideally be "opt-in". 

Can you make two changes here?

1. The `dest_addr` field should be #[cfg(any(target_os='darwin', target_os='netbsd'))] 
2. The cfg below should have the same conditions

Thanks!

Reply:
Looking more deeply -- it seems openbsd also supports this field.